### ISSUE TITLE
Weapons feel way too underpowered

### ISSUE THREAD

**Issue by ZeroAresIV** [DrMango14/Create-The_Factory_Must_Grow#80]
Everything From thermite grenades, pipe bombs and even the flamethrower feel like they're just for show. They're not really viable even against vanilla mobs. Imho the damage output should by multiplied by 3.

**Comment by WilliamIsMe1** [comment]
I'll see if I can rebalance them

**Comment by alperen-bircak** [comment]
Agree

**Comment by DrMango14** [comment]
you can burn a forest in seconds that doesnt seem underpowered to me

**Comment by ZeroAresIV** [comment]
Yeah you can burn down a forest. That’s not the point I’m addressing. The weapons themselves don’t do much physical damage, making a standard bow more effective in combat with players or even vanilla mobs.
